dts/bindings/ipc/zephyr,ipc-openamp-static-vrings.yaml

```diff
@@ -44,3 +44,10 @@ properties:
 
       When this property is missing a default priority of <0 PRIO_COOP> is
       assumed.
+
+  buffer-size:
```

Comment:
Shouldn't this be `zephyr,buffer-size` instead? Or is it really a hardware buffer?

Reply:
RPMSG buffet, so OpenAMP entity

Reply:
So still a software construct, right? Does it need the `zephyr` prefix then?

Reply:
no idea, you tell me :D yes, it a software construct but not of Zephyr itself.

Reply:
This is a bit confusing for me because this binding was introduced before we had the discussion on the mailing list that zephyr subsystem or driver software configuration in DT needs a `zephyr,` prefix.

We do have a `zephyr,priority` here, which is clearly a Zephyr related property. I don't know enough about OpenAMP to really follow you when you say "OpenAMP entity" and "software construct but not of Zephyr itself".

Reply:
It's just the size of the buffers that you want OpenAMP to use for TX and RX during IPC.